src/settings.js

```diff
@@ -217,6 +217,16 @@ var SIMD = 0;
 // Whether closure compiling is being run on this output
 var USE_CLOSURE_COMPILER = 0;
 
+// If set to 1, each asm.js/wasm module export is individually listed out to the
+// outside scope. This increases code size, and it is preferred to keep this disabled
+// when optimizing for smallest code size. If set to 0, module exports are copied
+// to parent scope via a short foreach loop, which can save a lot of space if
+// application has many exports from asm.js/wasm. However, setting this value to 0
+// can have dangerous behavior e.g. if using external minifier tools on the code
+// after the build, later, or if attempting to enclose the generated code in an
+// (anonymous) function/module scope.
+var DECLARE_ASM_MODULE_EXPORTS = 1;
```

Comment:
Pretty good, how about a little rephrasing though, like this?

> If set to 1, each asm.js/wasm module export is individually declared with a JavaScript "var" definition. This is the simple and recommended approach. However, this does increase code size (especiall if you have many such exports), which can be avoided in an unsafe way by setting this to 0. In that case, no "var" is created for each export, and instead a loop (of constant code size, with no hardcoded list of exports) writes all the exports received into the global scope. Doing so is dangerous since such modifications of the global scope can confuse external JS minifer tools, and also things can break if the scope the code is in is not the global scope (e.g. if you manually enclose them in a function scope).